packages/terra-clinical-result/src/common/utils.js

```diff
@@ -25,6 +25,34 @@ const checkTypeNumeric = (resultData) => {
   return isNumeric;
 };
 
+/*
+ * Returns a copy of the object with additional parameters with trimmed values and informational booleans
+ * @param {object} observationPropShape
+ * @return {object} is an object with added trimmed values and information.
+ */
+const createResult = (data) => {
```

Comment:
Should we rename to `santatizeResult`? I was a littl econfused when reading through Blood pressure and had to back track a bit to understand the difference between this object and the original

Reply:
Renamed https://github.com/cerner/terra-clinical/pull/648/commits/bcd9623974696cc69c750cf77514f41a424ac509#diff-72ca9008b342e1caa5acc1466dfa7f20R33